packages/core/src/js/feedback/FeedbackWidget.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable max-lines */
 import type { SendFeedbackParams, User } from '@sentry/core';
-import { captureFeedback, getCurrentScope, getGlobalScope, getIsolationScope, lastEventId, logger } from '@sentry/core';
+import { captureFeedback, debug,getCurrentScope, getGlobalScope, getIsolationScope, lastEventId } from '@sentry/core';
```

Comment:
not sure how this didnt trigger an error
```suggestion
import { captureFeedback, debug, getCurrentScope, getGlobalScope, getIsolationScope, lastEventId } from '@sentry/core';
```